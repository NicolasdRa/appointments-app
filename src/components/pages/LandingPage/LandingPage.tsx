import { Box, Hidden } from '@material-ui/core';

import { Header } from '../../ui/Header/Header';
import { HeroSection } from '../../ui/HeroSection/HeroSection';
import { BlurbSection } from '../../ui/BlurbSection/BlurbSection';

import { Footer } from '../../ui/Footer/Footer';

import { BottomNav } from '../../ui/BottomNav/BottomNav';

export const LandingPage = () => {
  return (
    <Box>
      <Header />
      <HeroSection />
      <BlurbSection />
      <Footer />
      <Hidden mdUp>
        <BottomNav />
      </Hidden>
    </Box>
  );
};
