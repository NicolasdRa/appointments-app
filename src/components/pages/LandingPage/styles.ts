import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';

export const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    main: { width: '100vw' },
  }),
);
