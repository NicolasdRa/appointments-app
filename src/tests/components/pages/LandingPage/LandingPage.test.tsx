import { LandingPage } from '../../../../components/pages/LandingPage/LandingPage';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

describe('tests on <LandingPage/>', () => {
  test('should render <LandingPage/> correctly', () => {
    const wrapper = shallow(<LandingPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
