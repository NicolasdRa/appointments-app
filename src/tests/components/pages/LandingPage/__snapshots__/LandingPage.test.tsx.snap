// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <LandingPage/> should render <LandingPage/> correctly 1`] = `
<Styled(MuiBox)>
  <Header />
  <HeroSection />
  <BlurbSection />
  <Footer />
  <Hidden
    mdUp={true}
  >
    <BottomNav />
  </Hidden>
</Styled(MuiBox)>
`;
